apiary.apib

```diff
@@ -159,6 +159,17 @@ Lists all tokens in the project.
                     "isMasterToken": true,
                     "canManageBuckets": true,
                     "canManageTokens": false,
+                    "canReadAllFileUploads": false,
+                    "canPurgeTrash": false,
+                    "canUseDirectAccess": false,
+                    "expires": null,
+                    "isExpired": false,
+                    "isDisabled": false,
+                    "dailyCapacity": 5,
+                    "creatorToken": {
+                        "id": 1,
+                        "description": "dev@keboola.com"
+                    },
```

Comment:
Tady by to být nemělo, jedná se o master token uživatele a ten to v reponse nemá. Je to zadané pouze u tokenů které někdo vytvořil pře Storage API

Reply:
Joo diky upravil som :)